### PR TITLE
mgr/cephadm: PEP8tify inventory.py

### PR DESCRIPTION
**Pull request by sebastian-philipp**
Signed-off-by: Sebastian Wagner <sebastian.wagner@suse.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: src/pybind/mgr/cephadm/inventory.py [hunk 1]
```diff
@@ -172,9 +173,9 @@ def __init__(self, mgr):
         self.osdspec_previews = {}     # type: Dict[str, List[Dict[str, Any]]]
         self.networks = {}             # type: Dict[str, Dict[str, List[str]]]
         self.last_device_update = {}   # type: Dict[str, datetime.datetime]
-        self.daemon_refresh_queue = [] # type: List[str]
-        self.device_refresh_queue = [] # type: List[str]
-        self.osdspec_previews_refresh_queue = [] # type: List[str]
+        self.daemon_refresh_queue = []  # type: List[str]
+        self.device_refresh_queue = []  # type: List[str]
+        self.osdspec_previews_refresh_queue = []  # type: List[str]
```
Comment (mgfritch) [inline src/pybind/mgr/cephadm/inventory.py:178]: same as https://github.com/ceph/ceph/pull/36523#discussion_r476443833?

**Comment by sebastian-philipp** [comment]
green: https://pulpito.ceph.com/swagner-2020-08-27_07:38:41-rados:cephadm-wip-swagner2-testing-2020-08-26-1329-distro-basic-smithi/

File: src/pybind/mgr/cephadm/inventory.py (hunk 1 above)
Comment (sebastian-philipp) [reply src/pybind/mgr/cephadm/inventory.py:178]: you're right and I was wrong. wdyt of merging it anyway?

Comment (mgfritch) [reply src/pybind/mgr/cephadm/inventory.py:178]: eh there's not really a right way, just curious if you wanted to keep the alignment. I'm aok with this.

**Pull request merged**